### PR TITLE
mtx: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/mtx/index.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export * as commands from './commands/index.js';
 export * as message from './message/index.js';
+export {default as CommandBinaryBuffer} from './utils/CommandBinaryBuffer.js';
```
Comment (DarkPark) [inline src/mtx/index.ts:3]: in utils should be index.js and here all utils should be exported

and constants

it's in my next mr

File: src/mtx/commands/downlink/setOpParams.ts [hunk 2]
```diff
@@ -0,0 +1,374 @@
+/**
+ * Downlink command to set device operator parameters.
+ *
+ * @example
+ * ```js
+ * import * as setOpParams from 'jooby-codec/mtx/commands/downlink/setOpParams.js'; * import {CommandBinaryBuffer} from 'jooby-codec/mtx/index.js';
+ *
+ * const parameters = {
```
Comment (DarkPark) [inline src/mtx/commands/downlink/setOpParams.ts:8]: there was getDefaultOpParams or something like this in buffer

maybe we can remove all this copy-paste?

File: src/mtx/commands/downlink/setSpecialDay.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import {READ_WRITE} from '../../constants/accessLevels.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer, ISpecialDay} from '../../utils/CommandBinaryBuffer.js';
+
+
+interface ISetSpecialDayParameters extends ISpecialDay {
+    /**
+     * tariff table identifier
+     * (`0` - table `A+`, `1` – table `A-`)
+     */
+    tariffTable: types.TUint8;
```
Comment (DarkPark) [inline src/mtx/commands/downlink/setSpecialDay.ts:38]:
```suggestion
    tariffTable: types.TUint8;

```
?

File: src/mtx/index.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/mtx/index.ts]: code removed

File: src/mtx/commands/downlink/setOpParams.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/mtx/commands/downlink/setOpParams.ts]: copy-paste removed here and in uplink/getOpParams.ts

File: src/mtx/commands/downlink/setSpecialDay.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/mtx/commands/downlink/setSpecialDay.ts:38]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**